moveit_ros/move_group/src/move_group.cpp

```diff
@@ -202,27 +251,36 @@ int main(int argc, char** argv)
         debug = true;
         break;
       }
+    debug = true;
     if (debug)
-      ROS_INFO("MoveGroup debug mode is ON");
+      RCLCPP_INFO(LOGGER, "MoveGroup debug mode is ON");
     else
-      ROS_INFO("MoveGroup debug mode is OFF");
+      RCLCPP_INFO(LOGGER, "MoveGroup debug mode is OFF");
+
+    rclcpp::executors::MultiThreadedExecutor executor;
+    rclcpp::Node::SharedPtr monitor_node = rclcpp::Node::make_shared("monitor_node", opt);
```

Comment:
this monitor_node is fed into `planning_scene_monitor->startStateMonitor` to prevent subscribers from never receiving messages. we can wait for #230 to go in first and adjust accordingly.

Reply:
It would probably make sense to merge that one first and then adjust it here. Do you see any issues with the implementation in #230? Woudl be great to have a review that is based on an actual use case like this.

Reply:
#230 is merged